src/Controller/SearchController.php

```diff
@@ -729,13 +754,18 @@ private function prepareResults(TransformedPaginatorAdapter $searchResults, $cur
                 if (method_exists($searchResult, 'getStatus')) {
                     $status = $searchResult->getStatus();
                 }
+                if (method_exists($searchResult, 'getItemId')) {
+                    $item = $itemService->getItem($searchResult->getItemId());
```

Comment:
Try using $searchResult here without the itemService

Reply:
I've not been successfull to use `$searchResult` instead of `$item`. I f I do so, I get:
```
Argument 1 passed to App\Utils\ReaderService::cachedReadStatusForItem() must be an instance of cs_item, instance of App\Entity\Discussions given, called in /var/www/html/src/Controller/SearchController.php on line 762
```